styles/books/chemistry/book.scss

```
@@ -2,6 +2,8 @@
 @import '../../templates/template1/design.scss';
 @import '../../templates/template1/settings.scss';
 
+@include use('CommonStyles', "CommonBookStuff");
```

```suggestion
@include use('RootTypography', "common:::RootTypography");
```
mostly to accomodate the namespaced convention